subsys/net/ip/ipv6.c

```diff
@@ -46,6 +46,12 @@
  */
 #define MAX_REACHABLE_TIME 3600000
 
+/* IPv6 minimum link MTU specified in RFC 8200 section 5
+ * Packet Size Issues
+ */
+#define MIN_MTU 1280
```

Comment:
The patch is fine, it is just that the name of these defines need to be more clear so that we do not get confused what MTU we mean here as there can be several MTU in the system. There is already a define called NET_IPV6_MTU which is set to 1280 that should be used here directly or by setting "#define MIN_IPV6_MTU NET_IPV6_MTU", also use MAX_IPV6_MTU instead of MAX_MTU.

Reply:
I've made requested changes, new commit pushed.